boards/xtensa/intel_s1000_crb/intel_s1000_crb.dts

```diff
@@ -35,11 +44,53 @@
 
 &spi0 {
 	status = "ok";
+	#address-cells = <1>;
```

Comment:
You don't need this #address-cells (already set to 1 in dts/xtensa/intel_s1000.dtsi)

Reply:
Done